### PR TITLE
add code for query binary classifier knowledge distillation 

### PR DESCRIPTION
**Pull request by ljshou**

### CODE REVIEW

File: model_zoo/nlp_tasks/knowledge_distillation/conf_kdqbc_bilstmattn_cnn.json [hunk 1]
```diff
@@ -0,0 +1,191 @@
+{
+  "license": "Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.",
+  "tool_version": "1.1.0",
+  "model_description": "This model is used for knowledge distillation for query binary classifier",
```
Comment (woailaosang) [inline model_zoo/nlp_tasks/knowledge_distillation/conf_kdqbc_bilstmattn_cnn.json]: **for** knowledge distillation **for** query binary classifier? More  grammatical description is better.

Comment (ljshou) [reply model_zoo/nlp_tasks/knowledge_distillation/conf_kdqbc_bilstmattn_cnn.json]: @chengfx

Comment (ljshou) [reply model_zoo/nlp_tasks/knowledge_distillation/conf_kdqbc_bilstmattn_cnn.json]: @woailaosang  do you have any suggestions?

File: tools/AUC.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            label.append(int(line[label_index]))
+    return prediction, label
+            
+def calculate_AUC(prediction, label):
+    return roc_auc_score(label, prediction)
+
+def main(params):
+    prediction, label = read_tsv(params)
+    auc = calculate_AUC(prediction, label)
+    print("auc is ", auc)
```
Comment (woailaosang) [inline tools/AUC.py]: print("AUC is ", auc)

Comment (woailaosang) [reply tools/AUC.py]: filename : "calculate_AUC.py" is better than "AUC.py", I think.

File: model_zoo/nlp_tasks/knowledge_distillation/conf_kdqbc_bilstmattn_cnn.json (hunk 1 above)
Comment (woailaosang) [reply model_zoo/nlp_tasks/knowledge_distillation/conf_kdqbc_bilstmattn_cnn.json]: Code is very good! I have run it smoothly.
Only some suggestions about name, description.

File: tools/AUC.py (hunk 2 above)
Comment (ljshou) [reply tools/AUC.py]: thanks Zhijie. @chengfx  please modify. 


> filename : "calculate_AUC.py" is better than "AUC.py", I think.

thanks Zhijie. @chengfx  please modify.

Comment (chengfx) [reply tools/AUC.py]: > print("AUC is ", auc)

Done

Comment (chengfx) [reply tools/AUC.py]: > thanks Zhijie. @chengfx  please modify.
> 
> filename : "calculate_AUC.py" is better than "AUC.py", I think.
> 
> thanks Zhijie. @chengfx  please modify.

thanks @woailaosang and @ljshou, done

File: model_zoo/nlp_tasks/knowledge_distillation/conf_kdqbc_bilstmattn_cnn.json (hunk 1 above)
Comment (chengfx) [reply model_zoo/nlp_tasks/knowledge_distillation/conf_kdqbc_bilstmattn_cnn.json]: > for knowledge distillation for query binary classifier? More  grammatical description is better.

@ljshou and I think it is okay

**Pull request merged**